composer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "kpodemski/demodeprecated",
+  "description": "Demo of how to deal with a deprecated methods",
+  "homepage": "https://github.com/kpodemski/demodeprecated",
+  "license": "MIT",
```

Comment:
I dont know if we can have MIT modules. AFL is necessary when sold on Addons, but outside?